SPECS-EXTENDED/fwupd-efi/fwupd-efi.spec

```diff
@@ -0,0 +1,127 @@
+%global debug_package %{nil}
+Summary:        Firmware update EFI binaries
+Name:           fwupd-efi
+Version:        1.6
+Release:        2%{?dist}
+License:        LGPL-2.1-or-later
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://github.com/fwupd/fwupd-efi
+Source0:        https://github.com/fwupd/fwupd-efi/releases/download/%{version}/%{name}-%{version}.tar.xz
+# Tolerate a slightly older version of gnu-efi
+Patch1:         azure_linux_compat.patch
+BuildRequires:  gcc
+BuildRequires:  gnu-efi-devel
+BuildRequires:  meson
+BuildRequires:  pesign
+BuildRequires:  python3-pefile
+BuildRequires:  python3-uswid
+BuildRequires:  python3-cbor2
+BuildRequires:  python3-lxml
+
+# these are the only architectures supporting UEFI UpdateCapsule
+ExclusiveArch:  x86_64 aarch64
+
+%description
+fwupd is a project to allow updating device firmware, and this package provides
+the EFI binary that is used for updating using UpdateCapsule.
+
+%prep
+%autosetup -p1
+
+%build
+
+%meson \
+    -Defi_sbat_distro_id="azurelinux" \
+    -Defi_sbat_distro_summary="Microsoft" \
+    -Defi_sbat_distro_pkgname="%{name}" \
+    -Defi_sbat_distro_version="%{version}-%{release}" \
+    -Defi_sbat_distro_url="https://github.com/microsoft/azurelinux"
+
+%meson_build
+
+%install
+%meson_install
+
+# sign fwupd.efi loader
+%ifarch x86_64
+%global efiarch x64
+%endif
+%ifarch aarch64
+%global efiarch aa64
+%endif
+%global fwup_efi_fn %{buildroot}%{_libexecdir}/fwupd/efi/fwupd%{efiarch}.efi
+%pesign -s -i %{fwup_efi_fn} -o %{fwup_efi_fn}.tmp
+%define __pesign_client_cert fwupd-signer
+%pesign -s -i %{fwup_efi_fn}.tmp -o %{fwup_efi_fn}.signed
+rm -vf %{fwup_efi_fn}.tmp
+
+%files
+%doc README.md AUTHORS
+%license COPYING
+%{_libexecdir}/fwupd/efi/*.efi
+%{_libexecdir}/fwupd/efi/*.efi.signed
+%{_libdir}/pkgconfig/fwupd-efi.pc
+
+%changelog
+* Mon Oct 28 2024 Jocelyn Berrendonner <jocelynb@microsoft.com> - 1.6-2
```

Comment:
should the initial version be the same as the source (1.6-1)?

Reply:
Hmm, the "Release" field above is set to "2%{?dist}", right? :)

Reply:
Based on the latest information from the build team, I need to revert back to align the build number with Fedora's package version.

Reply:
Actually, Andrew clarified that my approach is correct. I increased the release filed to "2", so the changelog has an associated 1.6-2 version.